lib/tapioca/dsl/compilers/protobuf.rb

```diff
@@ -91,6 +91,7 @@ def decorate
               create_type_members(klass, "Key", "Value")
             else
               descriptor = T.unsafe(constant).descriptor
+              next if descriptor.nil?
```

Comment:
NIT: this block is not a loop, so should we use `break` instead? Or alternatively, add `nil` to the `case` statement below and do nothing.

Reply:
I'd change the `gather_constants` to get subclasses of `Google::Protobuf::AbstractMessage` if it is defined instead of checking if the descriptor is `nil`. We should not generate the rbi for that class since it is already in the gem rbi.

Reply:
Yes, what @rafaelfranca said is the right call. If there is a new superclass for all protobuf classes, then all we need to do is to gather its descendants.

Reply:
Is there a "clean" way to get all descendants in this case? I can't do the following because `::Google::Protobuf::AbstractMessage` is a private constant:

```ruby
abstract_message_descendants = descendants_of(::Google::Protobuf::AbstractMessage)
```

Reply:
Thanks for the feedback! I've updated the DSL compiler to grab descendants of `::Google::Protobuf::AbstractMessage`, but ignore the constant itself (as its RBI is already in the gem RBI).

Not quite sure about how to test that yet. 👀

Reply:
You just need to add:
```ruby
              refute_includes(gather_constants, "Google::Protobuf::AbstractMessage")
```
inside the `gather_constants` tests.